gen/src/write.rs

```diff
@@ -1615,6 +1622,40 @@ fn write_rust_vec_impl(out: &mut OutFile, key: NamedImplKey) {
     writeln!(out, "}}");
 }
 
+fn write_kj_own(out: &mut OutFile, key: NamedImplKey) {
```

Comment:
let's add a comment with the example of the code it generates? I assume you have figured it out by now how to get one?

Reply:
at the very least document that it generates 2 extern "C" functions: drop and get, etc. etc.